output/t320-basis-f.out

```diff
@@ -0,0 +1,15 @@
+CeedBasis: dim=2 P=6 Q=4
+        qref:	  0.20000000	  0.60000002	  0.33333334	  0.20000000	  0.20000000	  0.20000000	  0.33333334	  0.60000002
+     qweight:	  0.26041666	  0.26041666	 -0.28125000	  0.26041666
```

Comment:
I'm not sure why these values computed in Fortran are slightly different that the ones computed in C for grad and interp.

Reply:
Likely it's because the floating point literals you use to initialize the vectors are single precision. They are promoted to double precision only after computing the values. Look at `qref` and `qweight`.